Base/AppTelemetry.cs

```diff
@@ -86,7 +86,7 @@ public void ConfigLoaded(ConfigFile configFile)
             foreach (InputConfigItem item in inputConfigs)
             {
                 String key = "input." + item.Type;
-                if (item.ModuleSerial.Contains(Joystick.SerialPrefix))
+                if (item.ModuleSerial.Contains(MFJoystick.SerialPrefix))
```

Comment:
So I went back and remembered why I did this. The problem is:

```C#
        private readonly SharpDX.DirectInput.Joystick joystick;
```

In Joystic.cs It should be `Joystick` per the property naming conventions. But you can't have a property have the same name as the class it is in, and the class is already called `Joystick`. So I renamed the class I guess.

We can go the other way and rename the property. What would be something instead of `Joystick` to use?

Reply:
Undid this change and instead renamed the property to `DXJoystick` inside the class.